docs/openshift-ai/data-science-project/using-projects-the-rhoai.md

```diff
@@ -0,0 +1,117 @@
+
+**Note**: Not all fields are required.
+
+- Name
+
+- Description
+
+- Notebook image (Image selection)
+
+- Deployment size (Container size and Number of GPUs)
+
+- Environment variables
+
+- Cluster storage name
+
+- Cluster storage description
+
+- Persistent storage size
+
+- Data connections
+
+!!! tip "How to specify CPUs, Memory, and GPUs for your JupyterLab workbench?"
+    You have the option to select different container sizes to define compute
+    resources, including CPUs and memory. Each container size comes with pre-configured
+    CPU and memory resources.
+
+    Optionally, you can specify the desired **Number of GPUs** depending on the
+    nature of your data analysis and machine learning code requirements. However,
+    this number should not exceed the GPU quota specified by the value of the
+    "**OpenShift Request on GPU Quota**" attribute that has been approved for
+    this "**NERC-OCP (OpenShift)**" resource allocation on NERC's ColdFront, as
+    [described here](../../get-started/allocation/allocation-details.md#pi-and-manager-allocation-view-of-openshift-resource-allocation).
+
+    If you need to increase this quota value, you can request a change as
+    [explained here](../../get-started/allocation/allocation-change-request.md#request-change-resource-allocation-attributes-for-openshift-project).
+
+Once you have entered the information for your workbench, click **Create**.
+
+![Fill Workbench Information](images/tensor-flow-workbench.png)
+
+For our example project, let's name it "TensorFlow Workbench". We'll select the
+**Tensorflow** image, choose a **Deployment size** of **Small**, and allocate a
+**Cluster storage** space of **1GB**.
+
+!!! info "More About Cluster Storage"
+    Cluster storage consists of Persistent Volume Claims (PVCs), which are
+    persistent storage spaces available for storing your notebooks and data. You
+    can create PVCs directly from here and mount them in your workbenches as
+    needed. It's worth noting that a default cluster storage (PVC) is automatically
+    created with the same name as your workbench to save your work.
+
+After creating the workbench, you will return to your project page. It shows the
+status of the workbench as shown below:
+
+![Workbench and Cluster Storage](images/workbench-cluster-storage.png)
+
+Notice that under the status indicator the workbench is *Running*. But if there
+are any issues i.e. an "insufficient memory" error then a red exclamation appears
+under the status indicator and you can hover over that icon. Looking closely at
+the error message, it shows the details about the issue and you can resolve accordingly.
+
+When your workbench is ready, the state will change to *Running* and you can select
+"Open" to go to your environment:
+
+![Open JupyterLab Environment](images/open-jupyter-lab.png)
+
+!!! tip "How can I start or stop a Workbench?"
+    You can use this toggle to easily *start/stop* this environment later on.
```

Comment:
what toggle?

Reply:
This is a **Toggle Switch** under the "Status" section.